mujoco_ros/Findmujoco.cmake

```diff
@@ -1,4 +1,4 @@
-find_package(mujoco QUIET NO_MODULE)
+find_package(mujoco::mujoco QUIET NO_MODULE)
```

Comment:
Hm. This looks very wrong as well. I guess, this is never found, which is why the fallback is called anyway.
I guess you already have a recipe generated for mujoco. Could you please share this for further debugging...